content/en/apps/guides/data/on-boarding-data-integrity.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Best Practices for Data Integrity During On-Boarding"
+linkTitle: "Data Integrity in On-Boarding"
+weight: 15
+description: >
+    How to ensure training data ends up in training and production data ends up production.
+
+---
+
+{{% pageinfo %}}
+This best practices guide should not be followed verbatim.  Instead, it is important to review each suggestion and assess if your deployment can benefit from it.
+{{% /pageinfo %}}
+
+## Introduction
+
+When on-boarding CHWs or refreshing their training, it is important to both enable them to learn how to use the CHT using a real mobile device with a real CHT Android application on the exact same configuration as production will use. This will give them the confidence to correctly and safely use the app to do their first household visits. This guide assumes that CHWs will have both a training and production app installed at the same time, or the CHW will be instructed to go install and log into the production app when training has been completed.
```

Comment:
I keep waiting for part two of that first sentence because of the "important to both...."

Reply:
that sentence is too long as it is!! removed the "both" in 7fe4410